internal/handler/model_handler.go

```diff
@@ -336,3 +337,62 @@ func DeleteModel(w http.ResponseWriter, r *http.Request) {
 
 	//render.OK(w, r)
 }
+
+// UpdateModelTemplate godoc
+//
+//	@Id				UpdateModelTemplate
+//
+//	@Summary		Update the elastic template with the model
+//	@Description	Update the elastic template with the model
+//	@Tags			Models
+//	@Produce		json
+//	@Param			id	path	string	true	"Model ID"
+//	@Security		Bearer
+//	@Security		ApiKeyAuth
+//	@Success		200	"Status OK"
+//	@Failure		400	"Status Bad Request"
+//	@Router			/engine/models/updateTemplate/{id} [put]
+func UpdateModelTemplate(w http.ResponseWriter, r *http.Request) {
```

Comment:
To match with routes pattern, change:

```suggestion
//	@Router			/engine/models/update-template/{id} [put]
func UpdateModelTemplate(w http.ResponseWriter, r *http.Request) {
```

or

```suggestion
//	@Router			/engine/models/updatetemplate/{id} [put]
func UpdateModelTemplate(w http.ResponseWriter, r *http.Request) {
```